src/content/docs/apis/intro-apis/introduction-new-relic-apis.mdx

```diff
@@ -72,20 +72,77 @@ This document provides examples and reference information for our API endpoints.
   To use APIs and SDKs, as well as the rest of our [observability platform](https://one.newrelic.com), join the New Relic family! [Sign up](https://newrelic.com/signup) to create your free account in only a few seconds. Then ingest up to 100GB of data for free each month. Forever.
 </Callout>
 
+
+## Ingest APIs for core data types [#data-type-apis]
+
+We have [many solutions for reporting data](/docs/telemetry-data-platform/get-started/introduction-new-relic-data-ingest-apis-sdks), and our APIs are one of these ways. These APIs can be used directly, but they're also the underlying ingest route for many of our more curated tools, such as our [open source telemetry integrations](/docs/integrations/open-source-telemetry-integrations/get-started/introduction-new-relics-open-source-telemetry-integrations) and our [Telemetry SDKs](/docs/telemetry-data-platform/ingest-apis/telemetry-sdks-report-custom-telemetry-data). 
+
+<table>
+  <thead>
+    <tr>
+      <th style={{ width: "150px" }}>
+        Data type
+      </th>
+
+      <th>
+        Description
+      </th>
+    </tr>
+  </thead>
+
+  <tbody>
+    <tr>
+      <td>
+        [Trace API](/docs/apm/distributed-tracing/trace-api/introduction-new-relic-trace-api)
+      </td>
+
+      <td>
+        Send distributed tracing data to New Relic.
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        [Event API](/docs/insights/insights-data-sources/custom-data/insert-custom-events-insights-api)
+      </td>
+
+      <td>
+        Send event data to New Relic.
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        [Metric API](/docs/introduction-new-relic-metric-api)
+      </td>
+
+      <td>
+        Send metrics to New Relic from any source (including other telemetry monitoring services).
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        [Log API](/docs/enable-new-relic-logs-http-input)
+      </td>
+
+      <td>
+        Send your log data to New Relic.
+      </td>
+    </tr>
+  </tbody>
+</table>
+
 ## NerdGraph (GraphQL) [#graphql]
 
-[NerdGraph](/docs/apis/graphql-api/getting-started/introduction-new-relic-graphql-api) is New Relic's GraphQL-format API, an efficient and flexible query language that lets you request exactly the data you need, without over-fetching or under-fetching. NerdGraph is the preferred API for querying New Relic data and making a range of feature configurations. 
+NerdGraph is the preferred API for querying New Relic data and making a range of feature configurations. [NerdGraph](/docs/apis/graphql-api/getting-started/introduction-new-relic-graphql-api) is our GraphQL-format API: GraphQL is an efficient and flexible query language that lets you request exactly the data you need, without over-fetching or under-fetching. 
```

Comment:
Who prefers it? Maybe we can make this first sentence less passive....

"We recommend using the NerdGraph API to query New Relic data because it provides a wide range of feature configurations"... or something. I actually have no idea why it's preferred :D

Reply:
it's preferred over our old Insights query API, which is basically deprecated (although still used by some). And the REST API has more functionality in place for configuring New Relic but it's slowly being supplanted by NerdGraph.